src/js/components/search/filters/awardType/SecondaryAwardType.jsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * SecondaryAwardType.jsx
+ * Created by Emily Gullo 11/02/2016
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    subList: React.PropTypes.array
+};
+
+export default class SecondaryAwardType extends React.Component {
+
+
+    render() {
+        const secondaryAwardList = (this.props.subList.map((subList, index) =>
```

Comment:
Too many parens is actually counterproductive re: readability.
I would move the array iteration logic up to PrimaryAwardType to match how the list of PrimaryAwards is iterated through in its parent AwardType component. This component should just be responsible for rendering a _specific_ secondary award type, not the full list.